examples/2016-10-31/image_binary_media_types/template.yaml

```diff
@@ -0,0 +1,68 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  ImageType:
+    Type: String
+    Default: image~1gif
+Globals:
+  Api:
+    # Send/receive binary data through the APIs
+    BinaryMediaTypes:
+      # These are equivalent to image/gif and image/png when deployed
+      - image~1jpg
+      - image~1gif
+      - image~1png
+      - application~1octet-stream
+
+Resources:
+  MyApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      DefinitionBody:
+        swagger: "2.0"
+        info:
+          version: "2016-09-23T22:23:23Z"
+          title: "Simple Api"
+        basePath: "/none"
+        schemes:
+          - "https"
+        paths:
+          "/none":
+            get:
+              x-amazon-apigateway-integration:
+                httpMethod: POST
+                type: aws_proxy
+                contentHandling: CONVERT_TO_BINARY
```

Comment:
SAM should allow users to specify this value without having to author their own swagger doc. One of the primary conveniences of using SAM is not having to author your own swagger. I don't think we can say SAM supports binary media types if you have to add a swagger doc to your template just to add this flag.